components/org.wso2.carbon.identity.api.server.tenant.management/org.wso2.carbon.identity.api.server.tenant.management.common/src/main/java/org/wso2/carbon/identity/api/server/tenant/management/common/TenantManagementConstants.java

```diff
@@ -52,6 +63,12 @@ public enum ErrorMessage {
         ERROR_CODE_PARTIALLY_CREATED_OR_UPDATED("TM-60021",
                 "Tenant creation / update was completed with errors.",
                 "Tenant creation / update was completed with error: %s"),
+        ERROR_CODE_INVALID_FILTER_FORMAT("TM-60022", "Invalid format used for filtering.",
+                "Filter needs to be in the format <attribute>+<operation>+<value>."),
```

Comment:
How about following ?
Invalid filter format. , The filter must follow the format: <attribute>+<operation>+<value>.